pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -6,10 +6,12 @@
 from django.contrib.postgres.fields import JSONField
 from django.db import models
 
-from pulp_2to3_migration.app.models import Pulp2to3Content
+from pulp_2to3_migration.app.models import Pulp2to3Content, Pulp2Repository
```

Comment:
nitpick, when it's more than 1 import, we tend to put them on the multiple lines
```
from X import (
  A,
  B,
)
```